<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryManagementSystem</class>
 <widget class="QMainWindow" name="LibraryManagementSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>754</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LibraryManagementSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>325</width>
           <height>681</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #D4D4D4;</string>
         </property>
         <widget class="QLabel" name="name_project_Label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>321</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 900 18pt &quot;Arial&quot;;
color: #000001;
text-align: center;

</string>
          </property>
          <property name="text">
           <string>Library Management</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>200</y>
            <width>181</width>
            <height>401</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="dauSach_pushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
             </property>
             <property name="text">
              <string>Đầu Sách</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>book-solid.svg</normaloff>book-solid.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="thedocgia_pushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
             </property>
             <property name="text">
              <string>Thẻ Độc Giả</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>user-regular.svg</normaloff>user-regular.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="muontra_pushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
             </property>
             <property name="text">
              <string>Mượn Trả</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>rotate-solid.svg</normaloff>rotate-solid.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="luuFile_pushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
             </property>
             <property name="text">
              <string>Lưu</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>download-solid.svg</normaloff>download-solid.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>0</y>
           <width>661</width>
           <height>681</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #F2F2F2;
</string>
         </property>
         <widget class="QStackedWidget" name="stackedWidget_infor">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>661</width>
            <height>671</height>
           </rect>
          </property>
          <widget class="QWidget" name="page_dausach">
           <widget class="QTableView" name="tableView_dausach">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>130</y>
              <width>631</width>
              <height>438</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableView {
    gridline-color: #d3d3d3;
    background-color: #f9f9f9;
    alternate-background-color: #eaeaea;
    border: 1px solid #d3d3d3;
    color: black; /* Set default text color to black */
}

QHeaderView::section {
    background-color: #D4D4D4;
    color: black;
    font-weight: bold;
    padding: 4px;
    border: 1px solid #d3d3d3;
}

/* Selected cell style */
QTableView::item:selected {
    background-color: #cce5ff;
    color: black; /* Ensure selected text color is black */
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLabel" name="timKiemDuSach_Label">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>10</y>
              <width>101</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: black;
font-size:20px;
font-weight:600;</string>
            </property>
            <property name="text">
             <string>Tìm Kiếm :</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_timkiemds">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>20</y>
              <width>481</width>
              <height>24</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color:black;</string>
            </property>
           </widget>
           <widget class="QPushButton" name="inTheLoai_pushButton">
            <property name="geometry">
             <rect>
              <x>480</x>
              <y>60</y>
              <width>171</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
            </property>
            <property name="text">
             <string>In Thể Loại</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>590</y>
              <width>511</width>
              <height>48</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
             <property name="spacing">
              <number>100</number>
             </property>
             <item>
              <widget class="QPushButton" name="themSach_pushButton_2">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
               </property>
               <property name="text">
                <string>Thanh lý</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>Thanhly.png</normaloff>Thanhly.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="themSach_pushButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
               </property>
               <property name="text">
                <string>Thêm</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>plus-solid.svg</normaloff>plus-solid.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_docgia">
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>100</y>
              <width>531</width>
              <height>391</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableWidget {
        gridline-color: #d3d3d3;
        background-color: #f9f9f9;  /* Light background for table */
        alternate-background-color: #eaeaea;  /* Alternating row color */
        border: 1px solid #d3d3d3;  /* Border around the table */
    }

    /* Header style */
    QHeaderView::section {
        background-color: #d3d3d3;  /* Gray background for header */
        color: black;               /* Black text color */
        font-weight: bold;          /* Bold text */
        padding: 4px;               /* Padding inside header cells */
        border: 1px solid #d3d3d3;  /* Header cell borders */
    }

    /* Cell styles */
    QTableWidget::item {
        padding: 4px;               /* Padding inside cells */
        border: none;               /* No cell borders */
		color:black;
    }

    /* Selected cell style */
    QTableWidget::item:selected {
        background-color: #cce5ff;  /* Light blue background for selected cell */
        color: black;               /* Black text color for selected cell */
    }

</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <column>
             <property name="text">
              <string>Mã thẻ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Họ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tên</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phái </string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Trạng Thái</string>
             </property>
            </column>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>50</y>
              <width>181</width>
              <height>26</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <bold>true</bold>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">text-align: right;</string>
               </property>
               <property name="text">
                <string>      Sắp xếp:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sapXepDocGia_ComboBox">
               <property name="styleSheet">
                <string notr="true">border: 1px solid gray;
color: black;
</string>
               </property>
               <item>
                <property name="text">
                 <string>Mã số</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tên</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>530</y>
              <width>451</width>
              <height>48</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>150</number>
             </property>
             <item>
              <widget class="QPushButton" name="themDocGia_pushButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
               </property>
               <property name="text">
                <string>Thêm</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>plus-solid.svg</normaloff>plus-solid.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="xoaDocGia_pushButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
               </property>
               <property name="text">
                <string>Xóa</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>trash-solid.svg</normaloff>trash-solid.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_muontra">
           <widget class="QListView" name="danhSachDocGiaMuonQuaHan_listView">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>70</y>
              <width>331</width>
              <height>451</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid gray;
color: black;</string>
            </property>
           </widget>
           <widget class="QTableView" name="topTenMuonNhieuNhat_tableView">
            <property name="geometry">
             <rect>
              <x>400</x>
              <y>70</y>
              <width>221</width>
              <height>451</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid gray;
color: black;</string>
            </property>
           </widget>
           <widget class="QPushButton" name="muonSach_pushButton">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>580</y>
              <width>141</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
            </property>
            <property name="text">
             <string>Mượn Sách</string>
            </property>
           </widget>
           <widget class="QPushButton" name="traSach_pushButton">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>580</y>
              <width>111</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
            </property>
            <property name="text">
             <string>Trả Sách</string>
            </property>
           </widget>
           <widget class="QPushButton" name="timSach_pushButton">
            <property name="geometry">
             <rect>
              <x>400</x>
              <y>580</y>
              <width>131</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #1899D6;
    border: none;
    border-radius: 16px;
    padding: 13px 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    text-transform: uppercase;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}</string>
            </property>
            <property name="text">
             <string>Tìm Sách</string>
            </property>
           </widget>
           <widget class="QLabel" name="muonQuaHan_Label">
            <property name="geometry">
             <rect>
              <x>52</x>
              <y>19</y>
              <width>251</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">text-align: center;
color: black;</string>
            </property>
            <property name="text">
             <string>Độc giả mượn sách quá hạn</string>
            </property>
           </widget>
           <widget class="QLabel" name="muonNhieuNhat_Label">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>10</y>
              <width>241</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: black;
</string>
            </property>
            <property name="text">
             <string>10 cuốn sách được mượn nhiều nhất </string>
            </property>
           </widget>
           <widget class="QTableView" name="tableView_danhsachquahan">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>70</y>
              <width>331</width>
              <height>451</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrary_Manegement_System">
    <property name="title">
     <string>Library Manegement System</string>
    </property>
   </widget>
   <addaction name="menuLibrary_Manegement_System"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
